python/ml4ir/base/io/local_io.py

```diff
@@ -0,0 +1,251 @@
+import os
+import json
+import shutil
+import pandas as pd
+import gzip
+import sys
+import csv
+import glob
+import yaml
+import logging
+
+from ml4ir.base.io.file_io import FileIO
+
+from typing import Optional
+
+from io import StringIO  # type: ignore
+
+
+class LocalIO(FileIO):
+    """Class defining the file I/O handler methods for the local file system"""
+
+    def make_directory(self, dir_path: str, clear_dir: bool = False) -> str:
```

Comment:
`clear_dir` is an overloaded name in this implementation. There are both functions and variables defined with the same name. Can we consider changing one of the two? For example, this bool arg can be renamed to `remove_dir_content` or something else equally descriptive. `clear_dir_content` would also work I guess.

Reply:
Fixed.